### PR TITLE
Fix typo's within converter.py

### PR DESCRIPTION
**Pull request by theCyberTech**
Fiexed Type errors

attemps > attempts

### CODE REVIEW

File: src/crewai/utilities/converter.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def _create_instructor(self):
 
         inst = Instructor(
             llm=self.llm,
-            max_attemps=self.max_attemps,
+            max_attemps=self.max_attempts,
```
Comment (pythonbyte) [inline src/crewai/utilities/converter.py:67]: I think we should check the "parameter" of the Instructor class max_attemps

Comment (theCyberTech) [reply src/crewai/utilities/converter.py:67]: could not find reference to max_attemps anywhere else apart from tools_usage.py which has now been updated an commited to this PR

File: src/crewai/utilities/converter.py [hunk 2]
```diff
@@ -22,7 +22,7 @@ class Converter(BaseModel):
     llm: Any = Field(description="The language model to be used to convert the text.")
     model: Any = Field(description="The model to be used to convert the text.")
     instructions: str = Field(description="Conversion instructions to the LLM.")
-    max_attemps: Optional[int] = Field(
+    max_attempts: Optional[int] = Field(
         description="Max number of attemps to try to get the output formated.",
```
Comment (gvieira) [inline src/crewai/utilities/converter.py:26]: Same typo here, right!?

Comment (gvieira) [reply src/crewai/utilities/converter.py:26]: Also `formated`.

**Pull request closed without merging**